shell/platform/android/io/flutter/app/FlutterApplication.java

```diff
@@ -6,7 +6,9 @@
 
 import android.app.Activity;
 import android.app.Application;
+import android.content.Context;
 import androidx.annotation.CallSuper;
+import com.google.android.play.core.splitcompat.SplitCompat;
```

Comment:
This can't be here anymore right? This will crash for legacy users that still use FlutterApplication if they don't supply the runtime dependencies themselves.

Reply:
Oh yeah, meant to remove this and the `attachBaseContext` override below

Reply:
Devs should be able to achieve this with https://developer.android.com/guide/playcore/play-feature-delivery#declare_splitcompatapplication_in_the_manifest